WordPress/WordPressTest/WPAnalyticsTrackerAutomatticTracksTests.m

```diff
@@ -0,0 +1,61 @@
+#import <Foundation/Foundation.h>
+#import <XCTest/XCTest.h>
+#import "WPAnalyticsTrackerAutomatticTracks.h"
+#import <Automattic-Tracks-iOS/TracksService.h>
+#import <OCMock/OCMock.h>
+
+@interface WPAnalyticsTrackerAutomatticTracksTests : XCTestCase
+
+@property (nonatomic, strong) WPAnalyticsTrackerAutomatticTracks *subject;
+@property (nonatomic, strong) id serviceMock;
+
+@end
+
+@interface WPAnalyticsTrackerAutomatticTracks (Testing)
+
+@property (nonatomic, strong) TracksService *tracksService;
+@property (nonatomic, strong) TracksContextManager *contextManager;
+
+@end
+
+@implementation WPAnalyticsTrackerAutomatticTracksTests
+
+- (void)setUp {
+    [super setUp];
+
+    self.subject = [[WPAnalyticsTrackerAutomatticTracks alloc] init];
+    self.serviceMock = OCMStrictClassMock([TracksService class]);
+    self.subject.tracksService = self.serviceMock;
+    self.subject.contextManager = nil;
+}
+
+- (void)tearDown {
+    [super tearDown];
+    
+    self.subject = nil;
+    self.serviceMock = nil;
+}
+
+- (void)testVerifyTracksNamesMappings {
+    for (NSUInteger x = 0; x <= WPAnalyticsStatTwoFactorSentSMS; ++x) {
```

Comment:
Thinking about this.  I can see new items added after `WPAnalyticsStatTwoFactorSentSMS` and forgetting to update this test. 
Something @jleandroperez (I think) has done is add a kind of "Count" enum as a convention, and referencing it whenever looping over all enums was needed.  It would require a pod change, tho so maybe out of scope here.